docs/docs/add-404-page.md

```diff
@@ -11,6 +11,18 @@ platforms default to using this as your 404 error page. If you're hosting your
 site another way, you'll need to set up a custom rule to serve this file for 404
 errors.
 
-When developing, Gatsby adds a default 404 page that overrides your custom 404
-page. But you can still preview your 404 page by clicking "Preview custom 404
-page" to verify that it's working as expected.
+Because Gatsby creates this page for you by default, there is no need to configure
+ it in your `gatsby-node.js` file. 
+
+When developing using `gatsby develop`, Gatsby uses a default 404 page that 
+overrides your custom 404 page. However, you can still preview your 404 page by 
+clicking "Preview custom 404 page" to verify that it's working as expected. This is
+useful when you're developing so that you can see all the available pages. 
+
+The photo below shows the default 404 page that Gatsby creates. As you can see, 
```

Comment:
```suggestion
The screenshot below shows the default 404 page that Gatsby creates. 
```
Making this a bit more inclusive without relying in sight!

Reply:
Love this feedback, thanks so much. Still new(ish) to development, working on accessibility!